server/sonar-web/src/main/js/apps/organizations/components/forms/ManageMemberGroupsForm.js

```diff
@@ -0,0 +1,155 @@
+/*
+ * SonarQube
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+// @flow
+import React from 'react';
+import Modal from 'react-modal';
+import { keyBy, pickBy } from 'lodash';
+import Checkbox from '../../../../components/controls/Checkbox';
+import { getUserGroups } from '../../../../api/users';
+import { translate, translateWithParameters } from '../../../../helpers/l10n';
+import type { Member } from '../../../../store/organizationsMembers/actions';
+import type { Organization, OrgGroup } from '../../../../store/organizations/duck';
+
+type Props = {
+  member: Member,
+  organization: Organization,
+  organizationGroups: Array<OrgGroup>,
+  updateMemberGroups: (member: Member, add: Array<string>, remove: Array<string>) => void
+};
+
+type State = {
+  open: boolean,
+  userGroups?: {},
+  loading?: boolean
+};
+
+export default class ManageMemberGroupsForm extends React.PureComponent {
+  props: Props;
+
+  state: State = {
+    open: false
+  };
+
+  openForm = () => {
+    if (!this.state.userGroups) {
+      this.loadUserGroups();
```

Comment:
`loadUserGroups` must be called each time the modal opens

otherwise there is a bug:

- go to the members page, open groups of a user
- go to the groups page, create a new group
- go back to the members page, open groups of the same user => you don't see just created group